packages/components/modules/profiles/useCurrentProfile/__tests__/CurrentProfileProvider.test.tsx

```
@@ -0,0 +1,55 @@
+
+describe('CurrentProfileProvider', () => {
+  beforeEach(() => {
+    localStorage.clear()
+    jest.clearAllMocks()
+  })
+
+  const loadPreStoredData = (profile: MinimalProfile) => {
+    setCookie(CURRENT_PROFILE_KEY, profile, { stringfyValue: true })
+  }
+
+  it.only('should get the user from local storage and not trigger the user profile fetch', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)